.gitignore

```diff
@@ -2,10 +2,10 @@
 .idea/
 .buildpath
 .settings/
+.idea/
```

Comment:
It's in the 1st line. This could be removed.

Reply:
@jbout should be fixed into develop later